Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -646,6 +647,114 @@ s32 ES::DIVerify(const IOS::ES::TMDReader& tmd, const IOS::ES::TicketReader& tic
   return IPC_SUCCESS;
 }
 
+constexpr u32 FIRST_PPC_UID = 0x1000;
+
+ReturnCode ES::CheckStreamKeyPermissions(const u32 uid, const u8* ticket_view,
+                                         const IOS::ES::TMDReader& tmd) const
+{
+  const u32 title_type = tmd.GetTitleType();
+  // Only allow using this function with some titles (WFS titles).
+  // The following is the exact check from IOS. Unfortunately, other than knowing that the
+  // title type is what IOS checks, we don't know anything about the constants used here.
+  if ((!(title_type & 4) && ~(title_type >> 5) & 1) || (title_type & 0x24) == 36)
```

Comment:
Done. I went one step further and replaced the constants by named values in an enum. We still don't know what 0x4 represents, but at least now the conditions should be more obvious.